core/remotes/docker/fetcher.go

```diff
@@ -444,10 +444,11 @@ func (r dockerFetcher) open(ctx context.Context, req *request, mediatype string,
 	if chunkSize < minChunkSize {
 		parallelism = 1
 	}
-	log.G(ctx).WithField("parallelism", parallelism).
+	log.G(ctx).WithField("initial_parallelism", r.performances.MaxConcurrentDownloads).
+		WithField("parallelism", parallelism).
 		WithField("chunk_size", chunkSize).
 		WithField("offset", offset).
-		Debug("fetching layer")
+		Info("fetching layer")
```

Comment:
Keep this as Debug, per connection logs can be so chatty

Reply:
Gotcha, moving back.
```suggestion
		Debug("fetching layer")
```